redbox-core/redbox/retriever/queries.py

```diff
@@ -68,47 +68,74 @@ def get_all(
     }
 
 
-def get_some(
-    embedding_model: Embeddings,
-    embedding_field_name: str,
+def get_metadata(
     chunk_resolution: ChunkResolution | None,
     state: RedboxState,
 ) -> dict[str, Any]:
-    vector = embedding_model.embed_query(state["request"].question)
+    query_filter = build_query_filter(
+        selected_files=state["request"].s3_keys,
+        permitted_files=state["request"].permitted_s3_keys,
+        chunk_resolution=chunk_resolution,
+    )
 
+    return {
+        "_source": {"excludes": ["*embedding", "text"]},
+        "query": {"bool": {"must": {"match_all": {}}, "filter": query_filter}},
+    }
+
+
+def build_document_query(
+    query: str,
+    query_vector: list[float],
+    embedding_field_name: str,
+    ai_settings: AISettings,
+    permitted_files: list[str],
+    selected_files: list[str] | None = None,
+    chunk_resolution: ChunkResolution | None = None,
+    adjacent: list[Document] | None = None,
+) -> dict[str, Any]:
+    """Builds a an Elasticsearch query that will return documents when called.
+
+    Searches the document:
+        * Text, as a keyword and similarity
+        * Name
+        * Description
+        * Keywords
+
+    If given a list of documents in adjacent, will boost adjacent documents by
```

Comment:
The code for this func looks good but I think the docstring could do with a bit more narrative to describe what the gaussian adjacency code adds to the function.